omg/entities/player.py

```diff
@@ -118,17 +119,17 @@ def update(self, mouse_x, mouse_y, delta_time):
             self._update_crafted_skill_slots(new_skill)
 
             # Empty the element cash
-            self.to_be_combined_element_cash = []
+            self.to_be_combined_element_buffer = []
 
     def shoot(self, skill_name: str):
         """Shoot a projectile and inform the observers."""
-        mana_cost = COMBINED_ELEMENT_DICTIONARY[skill_name].mana_cost
+        self.crafted_skill._set_skill_attributes(skill_name)
```

Comment:
if you use this method here, then `_set_skill_attributes` is not a private method and should not start with an underscore